### ISSUE TITLE
Warning: Package 'result' has an incorrect structure

### ISSUE THREAD

**Issue by schneiderfelipe** [arnetheduck/nim-result#15]
When using `result` as a dependency to a project, I get

```
      Info: Dependency on result@>= 0.2.0 already satisfied
   Warning: Package 'result' has an incorrect structure. The top level of the package source directory should
contain at most one module, named 'result.nim', but a file named 'results.nim' was found. This will be an error
in the future.
      Hint: If this is the primary source file in the package, rename it to 'result.nim'. If it's a source file
required by the main module, or if it is one of several modules exposed by 'result', then move it into a
'result/' subdirectory. If it's a test file or otherwise not required to build the the package 'result.nim',
prevent its installation by adding `skipFiles = @["results.nim"]` to the .nimble file. See
https://github.com/nim-lang/nimble#libraries for more info.
```

(Related #11?)

**Comment by arnetheduck** [comment]
Yeah, that specific piece of nimble advice, like many other things nimble, is a bit hard to follow in a real-world project - in this case, it would needlessly break backwards compatibility, which may happen at some point in the future, but no point in breaking it now.

**Comment by arnetheduck** [comment]
https://github.com/arnetheduck/nim-results/pull/29
